src/yoti_common/anchor.processor.js

```diff
@@ -266,29 +374,59 @@ module.exports.AnchorProcessor = class AnchorProcessor {
   }
 
   /**
-   * @returns {Array}
+   * @returns {Object}
    */
   static getResultFormat() {
-    const resultData = [];
-    resultData.sources = [];
-    resultData.verifiers = [];
-    return resultData;
+    return this.getAnchorTypes().reduce((acc, current) => {
+      acc[current] = [];
+      return acc;
+    }, {});
   }
 
+  /**
+   * Map of anchor list keys to oids.
+   *
+   * @returns {Object}
+   */
   static getAnchorTypesMap() {
     return {
-      sources: '1.3.6.1.4.1.47127.1.1.1',
-      verifiers: '1.3.6.1.4.1.47127.1.1.2',
+      sources: ANCHOR_TYPES.SOURCE,
+      verifiers: ANCHOR_TYPES.VERIFIER,
+      unknown: ANCHOR_TYPES.UNKNOWN,
     };
   }
 
   /**
+   * Get anchor list key by type.
+   *
+   * @param {string} type
+   */
+  static getAnchorListKeyByType(type) {
+    const anchorTypesMap = this.getAnchorTypesMap();
+    return Object.keys(anchorTypesMap)
+      .find(key => ANCHOR_TYPES[type] === anchorTypesMap[key]);
+  }
+
```

Comment:
I was going to say that this file has gotten quite large, so it might be worthwhile to split it up. _But_ we have deprecated methods now, so should be ok once they are removed.

Reply:
It also seems like some of the logic in `AnchorProcessor` could be moved to `YotiAnchor` and we may also want to rename the class to `Anchor` - could be safest to leave these as private classes for until the next major release.